### DISCUSSION TITLE
onedrive配置失败

### DISCUSSION THREAD

**Discussion by holob** [AlistGo/alist#391] [Q&A] [answered]
按照文档多次配置应用，在获取刷新令牌这一步失败。
报错提示如下![image](https://user-images.githubusercontent.com/94737046/150078905-40165dc5-7089-47fe-9e45-b4560d77d5c2.jpeg)

**Comment by xhofe** [comment, marked as answer]
https://stackoverflow.com/questions/57324634/aadsts700016-application-with-identifier-some-id-was-not-found-in-the-directo
